spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/GravitinoSparkConfig.java

```diff
@@ -10,6 +10,7 @@ public class GravitinoSparkConfig {
   private static final String GRAVITINO_PREFIX = "spark.sql.gravitino.";
   public static final String GRAVITINO_URI = GRAVITINO_PREFIX + "uri";
   public static final String GRAVITINO_METALAKE = GRAVITINO_PREFIX + "metalake";
+  public static final String GRAVITINO_SUPPORTS_ICEBERG = GRAVITINO_PREFIX + "supportsIceberg";
```

Comment:
referred from `SparkSession.builder.enableHiveSupport().getOrCreate().`, it seems enableIcebergSupport is more suitable.

Reply:
yes, this seems more suitable

Reply:
updated